eng/PatchConfig.props

```diff
@@ -27,5 +27,9 @@
     <PackagesInPatch>
     </PackagesInPatch>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.15' ">
+    <PackagesInPatch>
+    </PackagesInPatch>
+  </PropertyGroup>
```

Comment:
Going forward let's just reuse empty blocks in this file. There's no reason to build 2.1.11 again for example.

Don't do it in this PR but removing at least the following would be nice
``` msbuild
  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.11' ">
    <PackagesInPatch>
    </PackagesInPatch>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.12' ">
    <PackagesInPatch>
    </PackagesInPatch>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.14' ">
    <PackagesInPatch>
    </PackagesInPatch>
  </PropertyGroup>
```